Dashboard/app/js/templates/navData/monitoring-data.handlebars

```diff
@@ -157,9 +157,15 @@
                             {{else}}
                                 {{t _no_signature_found}}
                             {{/if}}
-                        {{else}}
+                        {{else}} {{#if view.isCaddisflyType}}
```

Comment:
Indentation of this section is not ideal. There is no easy way to _scan_ the code for the if` block